server/pulp/server/db/model/dispatch.py

```diff
@@ -489,3 +490,29 @@ def __init__(self, call_request, call_report):
         self.timestamp = dateutils.now_utc_timestamp()
         self.call_request_string = str(call_request)
         self.serialized_call_report = call_report.serialize()
+
+
+class CeleryBeatLock(Document):
+    """
+    Single document collection which gives information about the current celerybeat lock.
+    """
+
+    """
+        :param celerybeat_name:        string representing the celerybeat instance name
```

Comment:
This is a style thing, but I prefer the other way. Consider making these not to be vertically aligned. Each ending colon should have a single space and then the value.